pom.xml

```diff
@@ -52,6 +52,11 @@
             <artifactId>asm-tree</artifactId>
             <version>${asm.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.ow2.asm</groupId>
+            <artifactId>asm-commons</artifactId>
+            <version>${asm.version}</version>
+        </dependency>
```

Comment:
Is this dependency needed?

Reply:
sorry, they're not supposed to be changed.
i just reverted them back.